corehq/apps/cowin/tests.py

```diff
@@ -0,0 +1,124 @@
+import datetime
+import json
+import uuid
+
+from django.test import SimpleTestCase
+
+from mock import PropertyMock, patch
+
+from corehq.apps.cowin.repeater_generators import (
+    BeneficiaryRegistrationPayloadGenerator,
+    BeneficiaryVaccinationPayloadGenerator,
+)
+from corehq.apps.cowin.repeaters import (
+    BeneficiaryRegistrationRepeater,
+    BeneficiaryVaccinationRepeater,
+)
+from corehq.form_processor.models import CommCareCaseSQL
+from corehq.motech.models import ConnectionSettings
+from corehq.motech.repeaters.models import RepeatRecord
+
+
+class TestRepeaters(SimpleTestCase):
```

Comment:
Nice tests. I especially like that they're fast.

As mentioned earlier, it could be useful to test some mocked responses (happy, and errors) and how we deal with them.

Reply:
added test for a happy response, since only that is of interest here.